srslog_test.go

```diff
@@ -517,3 +517,39 @@ func TestConcurrentReconnect(t *testing.T) {
 		t.Error("timeout in concurrent reconnect")
 	}
 }
+
+func TestLocalConn(t *testing.T) {
+	messages := make([]string, 0)
+	conn := newTestLocalConn(&messages)
+
+	lc := localConn{conn: conn}
+
+	lc.writeString(nil, nil, LOG_ERR, "hostname", "tag", "content")
+
+	if len(messages) != 1 {
+		t.Errorf("should write one message")
+	}
+
+	if messages[0] != DefaultFramer(UnixFormatter(LOG_ERR, "hostname", "tag", "content")) {
+		t.Errorf("should use the unix formatter")
+	}
+}
+
+type testLocalConn struct {
+	messages *[]string
+}
+
+func newTestLocalConn(messages *[]string) testLocalConn {
+	return testLocalConn{
+		messages: messages,
```

Comment:
If you omit this comma, the code doesn't compile.

That's a Go thing. I guess they took the style of putting a comma at the end of these lines even when they're not the last item (to make diffs nicer later on) and they liked it so much they made it required.